app/javascript/react/screens/App/Overview/components/Cards/MigrationsInProgressCard/MigrationsInProgressCard.js

```diff
@@ -4,9 +4,36 @@ import { Grid, Icon, Spinner } from 'patternfly-react';
 import MigrationInProgressCard from './MigrationInProgressCard';
 
 class MigrationsInProgressCard extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isFetchingMigrationsInProgress: false
+    };
+  }
   componentDidMount() {
     const { fetchMigrationsInProgressAction } = this.props;
+    // fetch migrations in progress initially, then poll them
     fetchMigrationsInProgressAction();
+    this.pollingInterval = setInterval(fetchMigrationsInProgressAction, 10000);
+  }
+
+  componentWillReceiveProps(nextProps) {
+    // make the loading animation smooth - delay three seconds even if the backend is immediate
+    if (nextProps.isFetchingMigrationsInProgress) {
+      this.setState({
+        isFetchingMigrationsInProgress: true
+      });
+    } else {
+      setTimeout(() => {
+        this.setState({
+          isFetchingMigrationsInProgress: false
+        });
+      }, 3000);
+    }
+  }
+
```

Comment:
I think it will annoy users if we are constantly showing the wait cursor on each update. I don't believe we should change the cursor during auto-updates.

Reply:
good point - we can restrict this to **not** poll when a modal is open...does that work @jeff-phillips-18 ?

Reply:
(basically passing down the `mappingWizardVisible` and `planWizardVisible` state from Overview...)

Reply:
will update this now... give me 10 minutes ;)

Reply:
@priley86 Should the auto-updates be done silently - even when the modals are not visible?

i.e. no visual indication to the user about refreshing the migration status. (as in no spinners)

Reply:
i've made a change to prevent polling while modal wizards are open. LMK if this OK.